etl/src/main/java/org/pih/hivmigration/etl/sql/RegistrationMigrator.groovy

```diff
@@ -268,15 +295,51 @@ class RegistrationMigrator extends SqlMigrator {
                     uuid() as uuid
             from    hivmigration_patient_birthplace b,  hivmigration_registration_encounters r, hivmigration_patients p 
             where   b.source_patient_id = r.source_patient_id and r.source_patient_id = p.source_patient_id 
-                    and ( b.locality is not null or b.section is not null or b.commune is not null or b.department is not null);                                                                                                                                                                                                 
+                    and ( b.locality is not null or b.section is not null or b.commune is not null or b.department is not null);       
+                    
+                    
+            -- Add CIVIL STATUS
+            SET @civil_status_concept = (concept_from_mapping('PIH', 'CIVIL STATUS'));
+              
+            INSERT INTO obs (                    
+                    person_id, 
+                    encounter_id, 
+                    obs_datetime, 
+                    location_id, 
+                    concept_id,    
+                    value_coded,                
+                    creator, 
+                    date_created, 
+                    uuid)
+            select 
+                    p.person_id, 
+                    r.encounter_id, 
+                    r.encounter_date as obs_datetime, 
+                    1 as locationId, 
+                    @civil_status_concept as concept_id,  
+                    CASE s.civil_status 
+                        WHEN 'DIVORCED' THEN concept_from_mapping('PIH', 'DIVORCED') 
+                        WHEN 'MARRIED' THEN concept_from_mapping('PIH', 'MARRIED') 
+                        WHEN 'PARTNER' THEN concept_from_mapping('PIH', 'LIVING WITH PARTNER') 
+                        WHEN 'PLACE' THEN concept_from_mapping('PIH', 'LIVING WITH PARTNER') 
```

Comment:
Do we need to confirm/have we confirmed that "PLACE" means "Living with Partner"?

Reply:
Confirmed.